theme/snippets/nav.liquid

```diff
@@ -0,0 +1,39 @@
+<nav class="Nav none md:flex border-bottom-black text-nav uppercase sans-serif py_5 px_625 md:px1_25 flex justify-between">
+  <div>
+    <a class="{% if request.path == '/' %} Nav__button--active" {% endif %}" href="/" aria-label="{{ 'snippets.nav.links.index_link.aria_label' | t }}">{{ 'snippets.nav.links.index_link.text' | t }}</a>
+    <span>{{ 'snippets.nav.hyphen' | t }}</span>
+    {% for link in linklists.main-menu.links %}
+      <a class="{% if request.path == link.url %} Nav__button--active {% endif %}" href="{{ link.url }}">{{ link.title }}</a>{% unless link.title == linklists.main-menu.links.last.title %},{% endunless %}
```

Comment:
You can use `forloop.last` to write this a bit cleaner:

Instead of `{% unless link.title == linklists.main-menu.links.last.title %},{% endunless %}`, you can have `{% if forloop.last == false %},{% endif %}`.

And I would suggest putting it on a new line so it is easier to read.

Reply:
Ooh that's great, thank you!